Source/Urho3D/IO/Serializer.cpp

```diff
@@ -285,10 +287,24 @@ bool Serializer::WriteVariantData(const Variant& value)
         return WriteBuffer(value.GetBuffer());
 
         // Serializing pointers and custom values is not supported. Write null
+    case VAR_CUSTOM_STACK:
```

Comment:
Same here and in other places. `VAR_CUSTOM_*` is, logically, single type of Varinat (custom value)